### PR TITLE
Add recipe for unixodbc

### PR DESCRIPTION
**Pull request by xhochy**

### CODE REVIEW

File: recipes/unixodbc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  license_file: COPYING
+  summary: 'ODBC on non MS Windows platforms'
+  description: |
+    The unixODBC Project goals are to develop and promote unixODBC to be the
+    definitive standard for ODBC on non MS Windows platforms. This is to
+    include GUI support for both KDE and GNOME.
+
+extra:
+  recipe-maintainers:
+    - MathMagique
```
Comment (xhochy) [inline recipes/unixodbc/meta.yaml]: @MathMagique Is this ok for you?

Comment (MathMagique) [inline recipes/unixodbc/meta.yaml]: yes

**Comment by jakirkham** [comment]
LGTM. What do you mean about the OS X build, @wesm?

**Comment by wesm** [comment]
The build hadn't run when I commented. Merge away

**Pull request merged**